joomla/joomla-cms
# Remove from #__extensions useless columns: system_data and custom_data

Pull Request for Issue #14409

### Summary of Changes
Remove 2 columns from `#__extensions` table and all files.

1. Fix template copy functionality.
2. Fix postgresql installation, some plugins in extension table has missing column value.
3. Set correct PK value for template style: protostar in mysql
4. Set correct auto increment value for postgresql - copy template error

### Testing Instructions
1. Try to copy template style for protostar - before patch it does not work.

2. 
    a. Install dev-4.0 joomla branch and then apply patch and go to database fix button. Mentioned columns should be deleted from databases.
    b. Test installation of patched Joomla on mysql and/or postgresql.

3. Try to copy template style again - now it should work


### Expected result
Template copy works.

### Actual result
Error.

### Documentation Changes Required
Maybe. Two columns has been deleted from `#__extensions` table.
